src/utils/installPackage.js

```diff
@@ -0,0 +1,63 @@
+const {spawn} = require('child_process');
+const config = require('./config');
+
+function testYarn(options) {
```

Comment:
An alternative would be to check for a `yarn.lock` file in the `cwd`. It's probably a quicker check than spawning a whole process, and worst case we fall back to npm. This may also be desirable for the scenario that the user _has_ yarn but the project is using _npm_.

Reply:
Checking for a yarn file in comparison with spawning an instance is an insane speed difference i'll pivot back towards my initial setup, thank you for your input.

Reply:
Ah, didn't realize you'd started with that setup!